src/System.Security.Cryptography.Primitives/tests/CryptographicException.cs

```diff
@@ -17,7 +17,11 @@ public static void Ctor()
             Assert.NotNull(new CryptographicException().Message);
             Assert.Equal(message, new CryptographicException(message).Message);
             Assert.Equal(message + " 12345", new CryptographicException(message + " {0}", "12345").Message);
+#if netfx
```

Comment:
If you use `PlatformDetection` in src/Common/tests you can get rid of all the #ifdeffing and the need for a separate netfx configuration